js/packages/web/src/views/analytics/index.tsx

```diff
@@ -167,126 +167,127 @@ const rerun = async ({
   setUsersEngaged(engaged => ({ ...engaged, ...existingUsersEngaged }));
 };
 
-const MemoizedBar = React.memo(
-  (props: { sortedSales: number[]; mint: MintInfo }) => {
-    const histogrammedData: Record<number, number> = {
-      0: 0,
-      5: 0,
-      20: 0,
-      50: 0,
-      100: 0,
-      500: 0,
-      1000: 0,
-      10000: 0,
-    };
-    const asArray = [0, 5, 20, 50, 100, 500, 1000, 10000];
+const MemoizedBar = React.memo(function BarImpl(props: {
```

Comment:
GitHub diff really did a number on this, but all that really happened here is that I named the function passed to `React.memo` so as to satisfy the React ‘display name’ lint, and then Prettier indented everything differently.